github/apps_manifest.go

```diff
@@ -0,0 +1,54 @@
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+const (
+	mediaTypeAppManifestPreview = "application/vnd.github.fury-preview+json"
+)
+
+// AppConfig describes the configuration of a GitHub App.
+type AppConfig struct {
+	ID            *int64     `json:"id,omitempty"`
+	NodeID        *string    `json:"node_id,omitempty"`
+	Owner         *User      `json:"owner,omitempty"`
+	Name          *string    `json:"name,omitempty"`
+	Description   *string    `json:"description,omitempty"`
+	ExternalURL   *string    `json:"external_url,omitempty"`
+	HTMLURL       *string    `json:"html_url,omitempty"`
+	CreatedAt     *Timestamp `json:"created_at,omitempty"`
+	UpdatedAt     *Timestamp `json:"updated_at,omitempty"`
+	ClientID      *string    `json:"client_id,omitempty"`
+	ClientSecret  *string    `json:"client_secret,omitempty"`
+	WebhookSecret *string    `json:"webhook_secret,omitempty"`
+	PEM           *string    `json:"pem,omitempty"`
+}
+
+// AppManifestService contains methods for interacting with GitHub App
+// Manifests.
+// See
+// https://developer.github.com/apps/building-github-apps/creating-github-apps-from-a-manifest
+// for more details.
+type AppManifestService service
+
+// GetConfig completes the App manifest handshake flow for the given code.
+// See
+// https://developer.github.com/apps/building-github-apps/creating-github-apps-from-a-manifest/#3-you-exchange-the-temporary-code-to-retrieve-the-app-configuration
+func (s *AppManifestService) GetConfig(ctx context.Context, code string) (*AppConfig, *Response, error) {
```

Comment:
`GetConfig` sounds like it's just retrieving data, but what's happening here is actually the final step in the creation of the app.  So maybe `CreateAppFromManifest` or `CompleteAppManifest` or something like that?

Reply:
Done.